packages/thirdweb/src/wallets/in-app/core/users/findUsers.ts

```diff
@@ -0,0 +1,116 @@
+import type { ThirdwebClient } from "../../../../client/client.js";
+import { getThirdwebBaseUrl } from "../../../../utils/domains.js";
+import { getClientFetch } from "../../../../utils/fetch.js";
+import type { OneOf, Prettify } from "../../../../utils/type-utils.js";
+import type { Profile } from "../authentication/types.js";
+
+export type FindUserResult = {
+  userId: string;
+  walletAddress: string;
+  email?: string;
+  phone?: string;
+  createdAt: string;
+  profiles: Profile[];
```

Comment:
are you planning for this to be social profiles too? other i would just call these linkedAccounts i think to not confuse

Reply:
We call them profiles already (and in all the copy around that feature) so I figured linkedAccounts would be more confusing. Eventually we should consolidate these and social profiles